fix bug: AttributeError: 'Namespace' object has no attribute 'localfile' for fetch command

huakim opened the pull request.

mcepl commented on py2pack/__init__.py:
```diff
@@ -97,9 +97,9 @@ def _get_template_dirs():
     is important. The first found template from the first found dir wins!"""
     return filter(lambda x: os.path.exists(x), [
         # user dir
-        os.path.join(os.path.expanduser('~'), '.py2pack', 'templates'),
```
Why? Yes, I don’t expect support for Windows anytime soon, but defensive programming is always better.

Also, wouldn’t this be a nice opportunity to move from `~/.py2pack` to XDG-compliant `~/.config/py2pack`?

huakim replied:
yes, you right

huakim replied:
yes, i did

mcepl commented on py2pack/version.py:
```diff
@@ -16,6 +16,6 @@
 # limitations under the License.
 
 # this one is read by hatch or hatchling during build
-__version__ = "0.9.0"
```
Why do you feel the need to create new release here? What this has to do with `fix bug: AttributeError …`?

huakim replied:
because of new feautures?

mcepl replied:
Please, keep any PR only to the topic it purports to resolve. Creating new releases etc. is not a job of this PR.

mcepl reviewed (changes requested):
Also, please, add more to the description of this PR: I really don’t know what error it actually fixes (traceback? reproducer?) and thus I really don’t understand and cannot review how this commits helps.

huakim commented:
> Also, please, add more to the description of this PR: I really don’t know what error it actually fixes (traceback? reproducer?) and thus I really don’t understand and cannot review how this commits helps.

Steps to reproduce:
 - 1: write 'py2pack fetch py2pack' to command line
Expected result:
  py2pack tarball is downloaded
Current result: 
  traceback (most recent call):
 AttributeError: 'Namespace' object has no attribute 'localfile' for fetch command

The reason: 
  the object of Argparser class does not have local and localfile attributes, if we type 'py2pack fetch'
  i added workaround with try - except, but, later, workaround was deleted
  then i added one function 'fetch_local_data', which will work with attributes local and localfile, if none of these flags are provided, then 'fetch_local_data' function will call 'fetch_data' function

mcepl commented:
Please, rebase.

huakim commented:
> Please, rebase.

i added new branch 'main'
